ksp_plugin/interface_part.cpp

```diff
@@ -41,13 +41,15 @@ void __cdecl principia__PartIncrementIntrinsicForceWithPosition(
     Plugin* const plugin,
     PartId const part_id,
     XYZ const force_in_kilonewtons,
-    XYZ const position) {
+    XYZ const position,
```

Comment:
Maybe call this `force_position` to avoid ambiguities.

Reply:
`point_of_force_application`.